src/app/auth/auth.service.ts

```diff
@@ -29,11 +29,10 @@ export class AuthService {
   signUp(email, password) {
     this.afAuth.auth.createUserWithEmailAndPassword(email, password)
       .then(() => {
-        this.router.navigate(['/dashboard']);
         this.sendVerificationEmail();
       }).catch((error) => {
-        window.alert(error.message)
-      })
+      window.alert(error.message)
```

Comment:
Not part of this PR. 
In the long run I would get rid of this and create a nice little snackbar for this.
https://material.angular.io/components/snack-bar/examples